galley/mutations.py

```diff
@@ -69,3 +83,37 @@ def upsert_menu_data(args):
     mutation = build_upsert_mutation_query(args)
     response = make_request_to_galley(op=mutation)
     return validate_response_data(response)
+
+
+def build_update_mutation_query(args):
+    ids = []
+    if args["ids"] is not None and len(args["ids"]) > 0:
+        ids = {idx for idx in args["ids"] if type(idx).__name__ == "str"}
+        if len(ids) <= 0:
+            raise ValueError("non valid ID provided, all IDs must be strings")
+
+    mutation = Operation(Mutation)
+    bulk_input = BulkUpdateRecipeItemsInput(
+        ids=ids,
+        attrs=RecipeItemInput(args["attrs"])
+    )
+    mutation.bulkUpdateRecipeItems(input=bulk_input)
+    return mutation
+
+
+def update_recipe_item_data(args):
+    if "attrs" not in args.keys() or args["attrs"] is None:
+        raise ValueError("attrs property not provided")
+
+    if "ids" not in args.keys() or args["ids"] is None or len(args["ids"]) <= 0:
```

Comment:
```suggestion
    if not args.get("ids"):
```
This will check for both cases.

Reply:
This will check for all three conditions.